TUM Campus App/MenuTableViewCell.swift

```diff
@@ -12,10 +12,37 @@ class MenuTableViewCell: CardTableViewCell {
 
     override func setElement(_ element: DataElement) {
         if let menu = element as? CafeteriaMenu {
-            dishLabel.text = menu.name
-            priceLabel.text = ""
+            
+            //Dish Label
+            dishLabel.text = menu.details.nameWithEmojiWithoutAnnotations
+            
+            //Price Label
+            if let menuPrice = menu.price?.st {
+                let formatter = NumberFormatter()
+                formatter.numberStyle = .currency
+                formatter.locale = Locale(identifier: "de_DE")
```

Comment:
Oh ok. The formatter automatically adds the currency. I thought this was about "," vs "." in the formatting.